apps/angular/bug-effect-signal/src/app/app.component.ts

```diff
@@ -11,25 +11,42 @@ import { FormsModule } from '@angular/forms';
   imports: [FormsModule],
   selector: 'app-root',
   template: `
-    Show Dialog if one checkbox is checked
-    <input type="checkbox" [(ngModel)]="name" />
-    Name
-    <input type="checkbox" [(ngModel)]="age" />
-    Age
-    <input type="checkbox" [(ngModel)]="address" />
-    Address
+    <section class="flex gap-5">
+      <p>MacBook</p>
+      <p>1999,99 €</p>
+    </section>
+
+    <section>
+      <p>Extras:</p>
+
+      <div>
+        <input type="checkbox" [(ngModel)]="drive" />
+        +500 GB drive-space
+      </div>
+      <div>
+        <input type="checkbox" [(ngModel)]="ram" />
+        +4 GB RAM
+      </div>
+      <div>
+        <input type="checkbox" [(ngModel)]="gpu" />
+        Better GPU
+      </div>
+    </section>
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AppComponent {
-  name = model(false);
-  age = model(false);
-  address = model(false);
+  drive = model(false);
+  ram = model(false);
+  gpu = model(false);
 
   constructor() {
+    /* 
+      Explain for your junior team mate why this bug occurs ...
+    */
```

Comment:
This comment is not very useful to the challenge.

Reply:
To explain why this error occurs can be helpful to really understand it. But if you don't like it i can remove it.